app/src/main/res/layout/options_display.xml

```diff
@@ -90,6 +90,12 @@
                     app:options="@array/developer_options_msaa"
                     app:values="@array/developer_options_msaa_mode_values" />
 
+                <com.igalia.wolvic.ui.views.settings.RadioGroupSetting
```

Comment:
How about using `RadioGroupVSetting` here? Would that look better?

Reply:
Seems like no. This is how it looks with `RadioGroupVSetting`:
![Screenshot from 2024-10-22 22-58-19](https://github.com/user-attachments/assets/c8894879-1a9c-4552-a6d1-816281ec9b84)